### PR TITLE
Add traccar events documentation

### PR DESCRIPTION
**Pull request by lapy**
**Description:**

New function that imports traccar's events (and subsequently fires them in Home Assistant).

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22348

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Pherhaps some more description info for all the event variables?

**Comment by lapy** [comment]
Can I just add a link to the Traccar documentation?
https://www.traccar.org/documentation/events/

**Comment by klaasnicolaas** [comment]
Yes ofcourse, especially if it has extra added value 😉

**Comment by lapy** [comment]
Thanks @klaasnicolaas , done :)

File: source/_components/device_tracker.traccar.markdown [hunk 1]
```diff
@@ -64,6 +64,79 @@ monitored_conditions:
   description: Additional traccar computed attributes or device-related attributes to include in the scan.
   required: false
   type: list
+event:
+  description: Traccar events to include in the scan and fire within Home Assistant. *NOTE* For more info regarding Traccar events please refer to Traccar's documentation: https://www.traccar.org/documentation/events/.
```
Comment (klaasnicolaas) [inline source/_components/device_tracker.traccar.markdown]:
```suggestion
  description: "Traccar events to include in the scan and fire within Home Assistant. *NOTE* For more info regarding Traccar events please refer to Traccar's documentation: https://www.traccar.org/documentation/events/."
```

**Review by klaasnicolaas — DISMISSED** [review]
Will fix the build error 😉

**Comment by lapy** [comment]
Thanks, I always forget the quotes :P

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**